README.md

```diff
@@ -104,3 +104,5 @@ Credits for Images:
 * Blueberry: Made by TheJYKoder (original)
 * BlueberryBushFull: Made by TheJYKoder(original)
 * Strawberry: Created by Patrick Wang. Original.
+* GooseberryBushFull: http://www.quickcrop.co.uk
```

Comment:
Both images are encumbered :( Additionally the bush has a background, it should probably be a simple edited bush texture with a new color berry on it. Keep in mind the bush block will go through several stages and the "Full" stage needs to match style-wise with the earlier stages.